R/map.R

```diff
@@ -212,11 +213,7 @@ map_ <- function(.type,
 mmap_ <- function(.x, .f, .progress, .type, error_call, ...) {
 
   if (is.null(the$packages_installed)) {
```

Comment:
Maybe all this logic could move to `in_parallel()`?

Reply:
Moved in [8b48804](https://github.com/tidyverse/purrr/pull/1185/commits/8b4880480ba87fb05963229bfff2754b2d05fa4a). Note there's an outside chance someone could pass a crate created by `carrier::crate()` outside of purrr, and not have mirai installed. But I don't think it's worth worrying about.